execution_engine_testing/tests/src/test/regression/gh_1902.rs

```diff
@@ -90,7 +84,13 @@ fn should_not_charge_for_create_purse_in_first_time_bond() {
 
     let bond_amount = U512::from(BOND_AMOUNT);
     // This amount should be enough to make first time add_bid call.
-    let add_bid_payment_amount = U512::from(DEFAULT_ADD_BID_COST);
+    let add_bid_cost = builder
+        .get_engine_state()
+        .config()
+        .system_config()
+        .auction_costs()
```

Comment:
We could add `builder.get_auction_costs()` since it seems like a useful method to have.

Reply:
Added the suggested method